.github/workflows/build-and-push-assets.yml

```
@@ -99,11 +103,22 @@ jobs:
               .parse(process.env.ENV_VARS)
               .forEach(envVar => core.exportVariable(envVar.name, envVar.value));
 
+      - name: Set compile script
+        run: |
+          if [ "${{ inputs.MODE }}" == 'dev' ]; then
+           "COMPILE_SCRIPT=${{ inputs.COMPILE_SCRIPT_DEV }}" >> $GITHUB_ENV
+          elif [ "${{ inputs.MODE }}" == 'prod' ]; then
+           "COMPILE_SCRIPT=${{ inputs.COMPILE_SCRIPT_PROD }}" >> $GITHUB_ENV
+          elif [ "${{ inputs.MODE }}" == '' && ${{ contains(github.ref, 'refs/tags/') }} ]; then
```

Can we skip the check if `inputs.MODE` is empty? `dev` and `prod` values are handled in the checks above, and if it's something else or empty, we can just ignore it and only check for `github.ref`.
```suggestion
          elif [ ${{ contains(github.ref, 'refs/tags/') }} ]; then
```
(untested!)

---

This was just there for the sake of "completeness". It could be also possible that someone uses an invalid value for `inputs.MODE`. Hm..

---

>It could be also possible that someone uses an invalid value for `inputs.MODE`.

In this case, the `COMPILE_SCRIPT` will be `COMPILE_SCRIPT_PROD`. I think this is okay.